src/applications/claims-status/components/ClaimsDecision.jsx

```diff
@@ -34,6 +34,30 @@ const ClaimsDecision = ({ completedDate, showClaimLettersLink }) => (
         </p>
       )}
     </va-alert>
+
+    {!showClaimLettersLink && (
```

Comment:
I'm assuming this shouldn't be in this PR

Reply:
Yep, I told Kevin about that. I'll remove it.